metadeploy/routing.py

```diff
@@ -20,6 +26,6 @@
 application = ProtocolTypeRouter(
     {
         "http": get_asgi_application(),
-        "websocket": SessionMiddlewareStack(AuthMiddlewareStack(websockets)),
+        "websocket": metadeploy_stack(websockets),
```

Comment:
If we're getting rid of the `SessionMiddlewareStack` here, should `ChannelsCurrentSiteMiddleware` inherit from that instead of `BaseMiddleware`? IOW, will this break the places where we're accessing `self.scope["session"]` to get the `scratch_org_id`?

Reply:
Great question! The session logic is verified in multiple tests and it's working fine. In fact, `AuthMiddlewareStack` includes `SessionMiddleware` already, so I think the original code was running it twice